perfmetrics/scripts/vm_metrics/vm_metrics.py

```diff
@@ -312,7 +326,9 @@ def fetch_metrics(self, start_time_sec, end_time_sec, instance, period, test_typ
       row.append(end_time_sec)
       for metric in updated_metrics_list:
         row.append(metric.metric_point_list[i].value)
-      metrics_data.append(row)
+
+      # Skipping the first column as it duplicates the second column.
+      metrics_data.append(row[1:])
```

Comment:
FYI @raj-prince 
Ruchika had a [logic](https://github.com/GoogleCloudPlatform/gcsfuse/blob/experiments_dashboard/perfmetrics/scripts/vm_metrics/vm_metrics.py#L325) to append None values at the end, which seems unnecessary to me, as in our current code also we are not following the logic

Reply:
As we discussed offline, that was being added in the last for different fields, but skipping logic is required.